packages/tsort/tsort.1.0.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+synopsis: "Easy to use and user-friendly topological sort"
+description: """
+Easy to use and user-friendly topological sort.
+
+Example:
+```
+Tsort.sort [("foundation", []); ("walls", ["foundation"]); ("roof", ["walls"])]
+```
+"""
+maintainer: "daniil@baturin.org"
+authors: ["Daniil Baturin <daniil@baturin.org>"]
+homepage: "https://github.com/dmbaturin/ocaml-tsort"
+bug-reports: "https://github.com/dmbaturin/ocaml-tsort/issues"
+dev-repo: "git+https://github.com/dmbaturin/ocaml-tsort.git"
+license: "MIT"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "containers"
+  "dune"     {>= "1.5.1"}
```

Comment:
I'm a bit puzzled by this constraint. Where does it come from? I don't see any `dune-project` file at the root of your repository. Could you add it? It makes our (opam-repository maintainers) life infinity easier as future version of dune might break this package.

Reply:
I've just pushed a dune-project to dmbaturin/ocaml-tsort. Should I re-create a release with it in the tarball?

Reply:
If you can that'd be perfect (with the current `dune-project` file you have you will require to change the constraint to `{>= "1.9"}` though).

Reply:
Ok, I've re-created the release, and updated the opam file.
By future versions, you mean post-2.0? What's the best current practice in that regard now?